e2e/__tests__/__snapshots__/each.test.js.snap

```diff
@@ -127,9 +127,9 @@ FAIL __tests__/failure.test.js
   ✕ The word red contains the letter 'z'
   ✕ The word green contains the letter 'z'
   ✕ The word bean contains the letter 'z'
-  template table describe fails on all rows expected "a" == "b"
+  template table describe fails on all rows expected a == b
```

Comment:
These were more readable before, but I think people should add their own quotes if that makes sense for them (such as in this case) 🙂